src/NuGet.Config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="local package" value="Caliburn.Micro.Autofac.Packaging\" />
```

Comment:
Currently looking at [this thread](http://help.appveyor.com/discussions/problems/777-nuget-feed-for-project)